src/index.html

```diff
@@ -1,13 +1,281 @@
+
+          <div class="details__wrapper">
+            <article class="details__item">
+              <div class="details__images left-image-big">
+                <img src="images/d1.png" alt="details">
+                <img src="images/d2.png" alt="details">
+              </div>
+
+              <h3 class="details__name">Auto Unlock</h3>
+
+              <div class="details__description">
+                <p class="details__description--text">The app senses when you're nearby to unlock automatically. GPS tracking so you know where your bike is and can track it anytime.</p>
+              </div>
+            </article>
+
+            <article class="details__item">
+              <div class="details__images left-image-big">
+                <img src="images/d3.png" alt="details">
+                <img src="images/d4.png" alt="details">
+              </div>
+
+              <h3 class="details__name">Range & Integrated lights</h3>
+
+              <div class="details__description">
+                <p class="details__description--text">The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.</p>
+              </div>
+            </article>
+
+            <article class="details__item">
+              <div class="details__images right-image-big">
+                <img src="images/d5.png" alt="details">
+                <img src="images/d6.png" alt="details">
+              </div>
+
+              <h3 class="details__name">Hydraulic disc brakes & Lightweight</h3>
+
+              <div class="details__description">
+                <p class="details__description--text">Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.</p>
+              </div>
+            </article>
+
+            <button class="details__button button" onclick="window.location.href='#compare'">Explore</button>
+          </div>
+        </section>
+
+        <section class="main__contact contact" id="contact">
+          <h2 class="contact__title">Contact us</h2>
+          <div class="contact__form">
+            <form
+              action="/"
+              method="post"
+              class="contact__form--wrapper form"
+            >
+              <label for="name" class="d-none"></label>
+              <input
+                type="text"
+                name="name"
+                id="name"
+                class="input input-name"
+                placeholder="Name"
+                required
+                >
+
+              <label for="email" class="d-none"></label>
+              <input
+                type="email"
+                name="email"
+                id="email"
+                class="input input-email"
+                placeholder="Email"
+                required
+                >
```

Comment:
why would you need labels here?
they're not increasing the clickable area in this case - they're not actually functional.
But what they do - is adding the undesired gaps to your form grid:
<img width="989" alt="Monosnap Miami 2022-11-23 10-48-50" src="https://user-images.githubusercontent.com/39308717/203504731-2c06ccda-8ede-4f07-beca-53890777928c.png">

remove labels and see the difference